### PR TITLE
docs: add task guide to the sidebar

### PR DESCRIPTION
**Pull request by danyf90**
Resolves https://github.com/tuist/tuist/issues/YYY
Request for comments document (if applies):

### Short description 📝

> Describe here the purpose of your PR.

### How to test the changes locally 🧐

> Include a set of steps for the reviewer to test the changes locally.

### Checklist ✅

- [ ] The code architecture and patterns are consistent with the rest of the codebase.
- [ ] The changes have been tested following the [documented guidelines](https://docs.tuist.io/contributors/testing-strategy/).
- [ ] The `CHANGELOG.md` has been updated to reflect the changes. In case of a breaking change, it's been flagged as such.
- [ ] In case the PR introduces changes that affect users, the documentation has been updated.
- [ ] In case the PR introduces changes that affect how the cache artifact is generated starting from the `TuistGraph.Target`, the `Constants.cacheVersion` has been updated.

### CODE REVIEW

**Review by laxmorek — APPROVED** [review]

**Comment by ezraberch** [comment]
You need to update the title/slug in the header of the moved file:

https://github.com/tuist/tuist/blob/8d148be9389a644cc0a3d90e06e8ecf0f56f6727/projects/docs/docs/guides/task.md?plain=1#L1-L5

Links to the page also need to be changed.

**Comment by danyf90** [comment]
✅

File: projects/docs/docs/guides/task.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: tuist exec
```
Comment (ezraberch) [inline projects/docs/docs/guides/task.md:2]: Do you really want this to be the title which shows up in the taskbar?

File: projects/docs/docs/plugins/using-plugins.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ let config = Config(
 #### Tasks
 
 To use a task plugin, simply import the plugin in `Config.swift` and it will be automatically available by running `tuist my-plugin-task`.
-You can read more about tasks [here](/commands/task).
+You can read more about tasks [here](/guides/task).
```
Comment (ezraberch) [inline projects/docs/docs/plugins/using-plugins.md:83]:
```suggestion
You can read more about tasks [here](guides/task.md).
```

File: projects/docs/docs/plugins/creating-plugins.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ In order for Tuist to locate the templates for a plugin, they must be placed in
 
 ### Tasks
 
-Tasks represent arbitrary tasks which can be run via tuist. For more context, continue [here](/commands/task) where you will also find documentation for the `ProjectAutomation` framework.
+Tasks represent arbitrary tasks which can be run via tuist. For more context, continue [here](/guides/task) where you will also find documentation for the `ProjectAutomation` framework.
```
Comment (ezraberch) [inline projects/docs/docs/plugins/creating-plugins.md:58]: Using this form of link avoids future issues with versioned docs (#3945).

```suggestion
Tasks represent arbitrary tasks which can be run via tuist. For more context, continue [here](guides/task.md) where you will also find documentation for the `ProjectAutomation` framework.
```

**Review by ezraberch — CHANGES_REQUESTED** [review]

**Pull request merged**